ihontaryk/homework_2/fizz_buzz.py

```diff
@@ -0,0 +1,22 @@
+def fizz_buzz(number):
+    """
+    fizz_buzz function
+    """
+
+    if type(number) is not int:
+        raise TypeError
```

Comment:
more preferable way is 'isinstance' check

    if not isinstance(number, int):

Reply:
other way is at once try convering parameter into int, so if it is even string but is converted to int, it will be fine:

```

number = int(number)
```